src/holochainClient.js

```diff
@@ -131,14 +133,28 @@ async function initAndGetHolochainClient () {
     if (HOLOCHAIN_LOGGING) {
       console.log('🎉 Successfully connected to Holochain!')
     }
+    hcFlag = false
     return holochainClient
   } catch (error) {
     if (HOLOCHAIN_LOGGING) {
       console.log('😞 Holochain client connection failed -- ', error.toString())
     }
+    hcFlag = false
     throw (error)
   }
 }
+async function initAndGetHolochainClient () {
+  let counter = 0
+  while (hcFlag) {
+    counter++
+    await wait(100)
+    if (counter === 10) {
+      hcFlag = false
+    }
+  }
```

Comment:
isn't this equivalent to
```
if (hcFlag) {
  await wait(1000)
  hcFlag = false
}
```
?

Reply:
oh, I get it.

Going to think about if there's a more readable way to do this.